Dockerfile

```diff
@@ -33,8 +33,12 @@ RUN mkdir tessdata && cd tessdata \
     && find . -name '*.traineddata' -maxdepth 2 -exec cp {} /usr/share/tessdata \; \
     && cd .. && rm -r tessdata
 
-COPY dangerzone.py /usr/local/bin/
-RUN chmod +x /usr/local/bin/dangerzone.py
+COPY dangerzone/conversion/common.py /usr/local/bin/
+COPY dangerzone/conversion/pixels_to_pdf.py /usr/local/bin/
+COPY dangerzone/conversion/doc_to_pixels.py /usr/local/bin/
+RUN chmod +x /usr/local/bin/common.py
+RUN chmod +x /usr/local/bin/pixels_to_pdf.py
+RUN chmod +x /usr/local/bin/doc_to_pixels.py
```

Comment:
I think that these lines are not necessary, as we don't execute any of these files directly.

Reply:
Addressed [ab3ae26](https://github.com/freedomofpress/dangerzone/pull/437/commits/ab3ae26744476faffb759e79b1814b72e8dd29ae).